rules/S7495/python/metadata.json

```
@@ -0,0 +1,23 @@
+{
+  "title": "Direct dictionaries and sets comprehensions should be preferred over list comprehensions when creating dictionaries and sets respectively",
```

I feel that it isn't quite clear what a "direct dictionary" is. 
```suggestion
  "title": "Dictionary and set comprehensions should not be passed to \"dict\" and \"set\" respectively",
```

---

The title of the other rule might also work:

> Creation of collections with literals and comprehensions should not be wrapped in their type constructors